website/common/script/content/faq.js

```diff
@@ -1,4 +1,5 @@
 import t from './translation';
+import nconf from 'nconf';
```

Comment:
I removed it and reverted back to hardcoded values for this file for now. The challenge is that this file, being common, is used both by the client and the server and I'm having a hard time figuring out how to access values from config.json for each of those scenarios for a single file.